prov/efa/src/rxr/rxr_msg.c

```diff
@@ -632,11 +626,7 @@ ssize_t rxr_msg_multi_recv(struct rxr_ep *rxr_ep, const struct iovec *iov,
 	 * messages but will be used for tracking the application's buffer and
 	 * when to write the completion to release the buffer.
 	 */
-	rx_entry = rxr_ep_get_rx_entry(rxr_ep, iov, iov_count, tag,
-				       ignore, context,
-				       (rxr_ep->util_ep.caps &
-					FI_DIRECTED_RECV) ? addr :
```

Comment:
Is it safe to drop the FI_DIRECTED_RECV checks? `rxr_match_addr` does look for `FI_ADDR_UNSPEC`.

Reply:
Got it, makes sense. Dropping the matching logic in that scenario sounds reasonable to me.